### PR TITLE
Update lint setup

### PR DESCRIPTION
**Pull request by abdonrd**
Now will be easy add more linters like `polymer lint` or `stylelint`.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,18 +2,21 @@ language: node_js
...
     sources:
       - google-chrome
     packages:
       - google-chrome-stable
 node_js:
-  - node
-  - '6'
-before_script:
-  - npm install -g polymer-cli@next
+  - "node"
```
Comment (stramel) [inline .travis.yml]: I noticed a lot of Google repos are now using `stable` over `node` here.

File: .travis.yml [hunk 2]
```diff
@@ -2,18 +2,21 @@ language: node_js
...
+  - "6"
+install:
+  - npm install
+  - npm install -g polymer-cli
   - polymer install
+before_script:
+  - npm run lint
 script:
-  - xvfb-run npm test
+  - xvfb-run npm run test
```
Comment (stramel) [inline .travis.yml]: Are you adding `run` here for consistency?

File: package.json [hunk 3]
```diff
@@ -7,8 +7,9 @@
     "eslint-plugin-html": "^1.7.0"
   },
   "scripts": {
-    "lint": "eslint . --ext js,html --ignore-path .gitignore",
-    "test": "npm run lint && polymer test",
+    "lint": "npm run lint:javascript",
```
Comment (stramel) [inline package.json]: so would we can add `polymer lint` here maybe after they allow for no error on warnings?

File: .travis.yml (hunk 1 above)
Comment (abdonrd) [inline .travis.yml]: I'm based on the Polymer CLI: [Polymer/polymer-cli: `.travis.yml#L11-L13`](https://github.com/Polymer/polymer-cli/blob/master/.travis.yml#L11-L13)

@FredKSchott any thoughts?

File: .travis.yml (hunk 2 above)
Comment (abdonrd) [inline .travis.yml]: Yes, just for that. :)

File: package.json (hunk 3 above)
Comment (abdonrd) [inline package.json]: Exactly! Something like:

```json
[...]
"lint": "npm run lint:javascript && polymer lint",
"lint:javascript": "eslint . --ext js,html --ignore-path .gitignore",
[...]
```

File: .travis.yml (hunk 1 above)
Comment (abdonrd) [inline .travis.yml]: In the Travis docs only `node` appears:
[Travis docs | Specifying Node.js versions](https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Specifying-Node.js-versions)

Comment (stramel) [inline .travis.yml]: Works for me. Just had noticed a lot of them using stable. I couldn't find docs on it either.

**Pull request merged**